README.md

```diff
@@ -8,19 +8,19 @@ _Note that FontProof is very much a work-in-progress, so there's a lot that does
 
 ## Installation
 
-FontProof requires [SILE](https://github.com/simoncozens/sile). You'll need to install SILE according to that package's instructions, and then verify that it works by compiling the _examples/test.sil_ document.
+FontProof requires [SILE](https://github.com/simoncozens/sile). You'll need to install SILE according to that package's instructions, and then verify that it works by compiling the [_examples/test.sil_](https://github.com/sile-typesetter/sile/blob/master/examples/test.sil) document.
```

Comment:
I think I would drop this reference entirely. It isn't as relevant as it used to be, and could even be a hindrance. Since this was written SILE has been packaged for many platforms (Arch Linux, Void Linux, Ubuntu, Docker, Homebrew, Nix, etc.) that have their own testing mechanisms as part of the build process. Using those will make it easy to get up and running, and some of them don't even leave you with a mechanism to run that particular test file (e.g. Docker). It's only really relevant when building from source, which has it's own mechanisms to test with too. Why don't you just remove it...

(Also updating the URL here)

```suggestion
FontProof requires [SILE](https://github.com/sile-typesetter/sile). You'll need to install SILE according to that package's instructions.
```

Reply:
Indeed I just installed SILE via Homebrew and was having trouble locating the file. I'll remove the line, thanks!